doc/source/ray-observability/user-guides/profiling.md

```diff
@@ -0,0 +1,56 @@
+(profiling)=
+# Profiling
+Profiling is one of the most important debugging tools to diagnose performance, out of memory, hanging, or other application issues.
+Here is a list of common profiling tools you may use when debugging Ray applications. 
+- CPU profiling
+    - py-spy
+- Memory profiling
+    - memray
+- GPU profiling
+    - PyTorch Profiler
+- Ray Task / Actor profiling
```

Comment:
```suggestion
- Ray Task / Actor timeline tracing
```

Reply:
I'm not sure if it's tracing. When talking about tracing, to me, it's usually about profiling the time a request takes to go through different services.

Reply:
I think profiling is too general. Maybe call it timeline?

Reply:
Removed this title. Only kept the "Ray Task/Actor timeline"